sdk/daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Blinding.scala

```diff
@@ -121,11 +122,21 @@ object Blinding {
     }.toSeq
   }
 
-  /* Calculate the packages needed by a party to interpret the projection   */
+  /** Calculate the packages needed by each party in order to interpret the projection.
+    *
+    * This needs to include both packages needed by the engine at reinterpretation time
+    * and the originating contract package needed for contract model consistency checking.
+    *
+    * @param tx transaction whose packages are required
+    * @param contractPackages the contracts used by the transaction together with their creating packages
+    */
   def partyPackages(
       tx: VersionedTransaction,
       contractPackages: Map[ContractId, Ref.PackageId] = Map.empty,
```

Comment:
Given the ScalaDoc, I opt for removing the default arg, as it is usually wrong.

Reply:
This can be done as a follow up as removing it now would break the code drop.